### ISSUE TITLE
Swift 

### ISSUE THREAD

**Issue by sommanita** [codeforgso/lightning-talks#8]
We could do a few of these, each at different levels of expertise.

**Comment by braidn** [comment]
Swift like the language? My guess that a functional language/paradigms talk as a whole, minus language specific would be better for the entire group.

**Comment by nickmjones** [comment]
@braidn not sure I agree. Ordinarily, yes, but in this case there are a few of us who specifically want to learn more about Swift so we can spin up iOS apps for civic stuff.

**Comment by braidn** [comment]
Response noted. However, even now there are much easier ways to spin up an iOS app. React Native or even Rubymotion, even if you don't know the underlying language, are much easier to kickstart iOS apps

**Comment by sommanita** [comment]
You're welcome to add an issue that would cover a general approach. I think we need to remember that these "issues" are merely suggestions that we have as individuals. It might be true not everyone in the group would want to learn about Swift, but that is just as true for any other topic on the list.

**Comment by braidn** [comment]
Oh, they are just suggestions? Not topics for conversation and talking/figuring things out? My bad.

**Comment by sommanita** [comment]
Hi. We are free to have conversations. What I was trying to explain, kindly, was that this is simply an item on a list of talks. Swift is just one talk that I would like to see happen. If I end up organizing it and it's just me sitting there, that's totally cool. If everyone shows up, that's also cool. If this idea sparked some other ideas for you, that is great and I hope you add those topics to this list also. 

I will go ahead and admit that your response here seems incredibly argumentative. I am going to assume that wasn't your intention. Arguing in a public forum, on GitHub about items on a list of zero consequence is unnecessary. So I hope you do understand that no, no one is shutting you down from discussing things. It is simply that no one who adds anything to a list of lightning talks should have to defend their idea. This is just a list. Add to it, ask questions, but let's not argue about it.

**Comment by braidn** [comment]
Feel free to correct me here but, I have been rooted in the tech community here for quite a while. And a talk at the complexity level of Swift would be sort of... "over most folks heads" let's say. I do think these talks should be a functional gateway into people feeling free to be able to add value to the organization as a whole. This isn't accomplished by teaching a subject that isn't grok-able by the entirety of the group. 

With my comments above I simply tried to offer up suggestions to come to the end goal of "an IOS app". If there was context affixed to them, I apologize that you assigned the negative to them.

**Comment by nickmjones** [comment]
This is pretty much a non-apology. I've been "rooted in the tech community" for over 16 years, and I would still never want to suggest that any subject would be "over the heads" of 64 people I've never met. 

A big component of Code for America is the social contract that we uphold with one another in these discussions. We're not interested in turning this into reddit. We're also not interested in having any of our members make assumptions about the skill level of other members without basis. That was the core of my original objection, which you seemed to take umbrage with. 

If you want to help, help. If it's clear that other members have an interest in something that you don't, make a new suggestion. Don't just thread shit. 

Also, as I am also "rooted in the tech community" I can say with authority that RubyMotion sucks. And why do you not seem to think that React Native would be "over people's heads"? I have experience with that, and it's certainly not for beginners. 

The bottom line is I'm closing this issue. I'll reopen it for those who are interested. If that isn't you, just skip it. It's really pretty simple. I would also ask that you participate a bit more before making a blanket statement about this group and its members.
